Livecheckables/jdnssec-tools.rb

```diff
@@ -1,6 +1,6 @@
 class JdnssecTools
   livecheck do
-    url :head
-    regex(/^v?(\d+(?:\.\d+)+)$/)
+    url "https://github.com/dblacka/jdnssec-tools/releases/latest"
+    regex(%r{href=.+?/jdnssec-tools-v?(\d+(?:\.\d+)+)\.t}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["']}i)
```

An archive with this type of name may or may not be present in a given release, so it's more reliable to match the tag when checking the "latest" release page.

Reply:
I am actually debating on relying on the latest release (in the past, we did see the latest does not include the needed artifact either).

But I guess we can turn it into a upstream repo issue instead.